### ISSUE TITLE
pytorch warnings

### ISSUE THREAD

**Issue by woiza** [kaushaltrivedi/fast-bert#244]
I followed your notebook on multi-label classification:
fast-bert/sample_notebooks/new-toxic-multilabel.ipynb

and got the following warnings:

> pytorch/lib/python3.7/site-packages/torch/nn/parallel/_functions.py:61: UserWarning: Was asked to gather along dimension 0, but all input tensors were scalars; will instead unsqueeze and return a vector.
  warnings.warn('Was asked to gather along dimension 0, but all '

> pytorch/lib/python3.7/site-packages/torch/optim/lr_scheduler.py:224: UserWarning: To get the last learning rate computed by the scheduler, please use `get_last_lr()`.
  warnings.warn("To get the last learning rate computed by the scheduler, "

How can I fix them?
